src/widgets/Window.cpp

```diff
@@ -711,6 +719,13 @@ void Window::addMenuBar()
     // Window menu.
     QMenu *windowMenu = mainMenu->addMenu(QString("Window"));
 
+    // Window->Minimise item
+    QAction *minimiseWindow = windowMenu->addAction(QString("Minimise"));
+    minimiseWindow->setShortcuts({QKeySequence("Meta+M")});
```

Comment:
Is this standard behavior to have a `Window->Minimise` action on macos? This is probably a window manager's job to handle minimising a window and have a shortcut for it.

Reply:
A lot of application on macOS do have minimise action. (See examples below)

I have not been using macOS very long, and also trying to wrap my head around this code, so I am going off from what I can see from the different applications I am using. I could very well be wrong here.

Here is one from Safari browser:
<img width="1006" alt="image" src="https://github.com/Chatterino/chatterino2/assets/881987/6459738e-1dc8-4f0c-a7f6-173d7afec8d4">

Here is one from VSCode:
<img width="924" alt="image" src="https://github.com/Chatterino/chatterino2/assets/881987/32548d52-020f-4c28-8298-431f821b0736">